### PR TITLE
Add Tencent.WeType version 0.9.0.69

### PR DESCRIPTION
**Pull request by StarsbySea**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/108409)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Exception during executable launch operation System.AggregateException: One or more errors occurred. (The specified executable is not a valid application for this OS platform.)  ---
> System.ComponentModel.Win32Exception (216): The specified executable is not a valid application for this OS platform.

File: manifests/t/Tencent/WeType/0.9.0.69/Tencent.WeType.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Tencent.WeType
+PackageVersion: 0.9.0.69
+Installers:
+- InstallerUrl: https://wetype.wxqcloud.qq.com/app/win/WeTypeSetup_0.9.0.69.exe
+  Architecture: x86
```
Comment (stephengillie) [inline manifests/t/Tencent/WeType/0.9.0.69/Tencent.WeType.installer.yaml:8]:
```suggestion
  Architecture: x64
```

Comment (stephengillie) [reply manifests/t/Tencent/WeType/0.9.0.69/Tencent.WeType.installer.yaml:8]: Hi @StarsBySea,

`x86` packages are validated on a 32-bit OS, so I'm guessing this is a 64-bit application.

Comment (StarsbySea) [reply manifests/t/Tencent/WeType/0.9.0.69/Tencent.WeType.installer.yaml]: Thank you. You're right!

**Review by stephengillie — APPROVED** [review]

**Pull request merged**